### PR TITLE
MGMT-10256: remove assisted jobs for selected versions

### PR DESCRIPTION
**Pull request by osherdp**
Since we have another validation going on with [bump_ocp_releases](https://github.com/openshift-assisted/assisted-installer-deployment/blob/05514c359b1554d05a0ef7f34b011b2181ff32bd/tools/bump_ocp_releases.py) there's no need for us to validate assisted flow with every created payload for those versions.

It takes a lots a of of money to provision all those Equinix machines, and we've yet to find any regression using those kind of nightly jobs.

### CODE REVIEW

**Comment by osherdp** [comment]
/cc @gamli75 @adriengentil @eliorerz 
/hold
(waiting for rehearse jobs)

**Comment by osherdp** [comment]
All failures seem to be connected to the imagestream import incident, and not to this change

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml [hunk 1]
```diff
@@ -7,26 +7,6 @@ base_images:
...
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-installer-controller
-  assisted-service:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-service
   assisted-test-infra:
     name: ocm-2.6
     namespace: edge-infrastructure
```
Comment (eliorerz) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml:12]: What about test-infra?

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml [hunk 2]
```diff
@@ -7,26 +7,6 @@ base_images:
...
-    tag: assisted-installer-agent
-  assisted-installer-controller:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-installer-controller
-  assisted-service:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-service
   assisted-test-infra:
```
Comment (eliorerz) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml:10]: Same here, why do we need it?

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml [hunk 3]
```diff
@@ -3,26 +3,6 @@ base_images:
...
-    tag: assisted-installer-agent
-  assisted-installer-controller:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-installer-controller
-  assisted-service:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-service
   assisted-test-infra:
```
Comment (eliorerz) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml:6]: Same here

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml (hunk 1 above)
Comment (osherdp) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml:12]: live-iso jobs use it, so it's required

**Comment by osherdp** [comment]
trying to retest to see if the problem still occurs:
/retest

**Review by eliorerz — APPROVED** [review]

**Pull request merged**